locales/en-US/webextension.properties

```diff
@@ -58,7 +58,7 @@ genericErrorTitle = Whoa! Firefox Screenshots went haywire.
 genericErrorDetails = We’re not sure what just happened. Care to try again or take a shot of a different page?
 # Section for onboarding strings
 # Slide number 1:
-tourBodyIntro = Take, save, and share screenshots without leaving Firefox.
+tourBodyIntroServerless = Take screenshots without leaving Firefox.
```

Comment:
@ianb michelle wanted to review this string before we merge

Reply:
@johngruen Probably need that review today, or tomorrow at latest, to land this in mozilla-central before the end of week deadline

Reply:
The new string is per feedback from Meridel (who is working on this with us instead of Michelle)